packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/semanticdata/SemanticData.java

```diff
@@ -151,6 +174,20 @@ public static Builder newSemanticData() {
         return new Builder();
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Having `equals` and `hashCode` in `SemanticData` eases my life when computing transitive dependencies. This could have some consequences I do not anticipate though. If this is the case, I can rework the code so that we don't need it.